packages/engine-server/src/metadata/service.ts

```diff
@@ -65,6 +65,10 @@ type Metadata = Partial<{
    *
    */
   workspaceActivationContext: WorkspaceActivationContext;
+  /**
+   * Default Theme for Note Graph View
+   */
+  defaultGraphTheme?: GraphThemeEnum;
```

Comment:
nit: remove the word 'default' here - it's the user's _current_ theme that we're setting here.